src/Features/CSharp/Portable/EditAndContinue/StatementSyntaxComparer.cs

```diff
@@ -221,6 +221,15 @@ internal enum Label
             GroupClauseLambda,                // tied to parent
             QueryContinuation,                // tied to parent
 
+            TypeParameterList,                // tied to parent
```

Comment:
>TypeParameterList [](start = 12, length = 17)

perhaps add a comment above this line that the following are child nodes of local function statement and lambdas